docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,306 @@
 ---
-title: How to Contribute to Teleport's Documentation
-description: Follow this guide to get started contributing changes to Teleport's documentation
+title: How to contribute to Teleport documentation
+description: Follow this guide to learn how to contribute changes to Teleport documentation.
 ---
 
-This guide describes the general workflow for making documentation changes.
-It assumes you already have Node >=14 and `yarn` installed.
+We encourage everyone with an interest in improving Teleport documentation to contribute content, submit issues, and suggest changes.
+This guide describes the process for making changes to the documentation whether you are an internal or external contributor.
 
-## Step 1/4 Set up your local environment
+## Before you begin
 
-Clone the `gravitational/docs` repository via `git clone`:
+Before you begin, verify that your environment meets the following prerequisites:
 
-```code
-$ git clone https://github.com/gravitational/docs
-```
+* You have a code editor, a GitHub account, and experience using command-line programs, including `git` commands and command line options.
 
-Different versions of the documentation are organized into the `content`
-directory of `gravitational/docs`. Each subdirectory of `content` includes a git
-submodule for a different branch of `gravitational/teleport`.
+* You have Node.js, version 14.x or newer, installed. 
+   Run `node --version` to check whether Node is installed.
+   If you need to install or update Node, see [Installing Node.js](https://nodejs.org/en/download/package-manager) for instructions to download and install Node.js using a package manager.
 
-Navigate to the root of your local clone of the `gravitational/docs` repository
-and run the following command to populate the `content` directory:
+* You have the `yarn` package manager installed.
+   Run `yarn --version` to check whether `yarn` is installed. 
+   If you need to install `yarn`, run the command `npm install --global yarn` to add it to your local system.
 
-```code
-$ yarn git-update
-```
+## Step 1/5. Set up your local environment
 
-Next, navigate to the directory under `content` that corresponds to the latest version of Teleport.
+To set up a local environment for contributing to Teleport documentation:
 
-```code
-$ cd content/(=teleport.major_version=).x
-```
+1. Open a terminal shell on your computer.
 
-Switch to the master branch (or versioned branch for updates specific to previous versions):
+1. Clone the `gravitational/docs` repository by running the following command:
```

Comment:
Since we have a lot of commands in quick succession in this section, I think it would make sense to remove "by running the following command" after the first one, otherwise it gets a little distracting for me.

Reply:
I sort of like the consistency, but I get your point.